posydon/grids/io.py

```diff
@@ -413,7 +413,10 @@ def initial_values_from_dirname(mesa_dir):
     if "initial_mass" in dirname:                           # single-star grid
```

Comment:
A minor thing here. For v1 single star grids, there is no initial_z.
if "initial_z" not in dirname:
        variable_names = ["initial_mass"]
else:
        variable_names = ["initial_mass", "initial_z"]

Reply:
I have added a check for v1 in the single star case